src/base-container/components/default-layout/index.jsx

```diff
@@ -0,0 +1,3 @@
+export { default as DefaultLargeLayout } from './LargeLayout';
+export { default as DefaultMediumLayout } from './MediumLayout';
+export { default as DefaultSmallLayout } from './SmallLayout';
```

Comment:
unable to understand the location of these `'./LargeLayout'`. where do they exist?

Reply:
Inside the default-layout folder. Every layout has its own large, medium and small screen design
![Uploading Screenshot 2023-06-28 at 3.57.03 PM.png…]()